lib/whitehall_importer/import.rb

```diff
@@ -19,6 +19,10 @@ def call
       end
 
       ActiveRecord::Base.transaction do
+        document_import.update!(
+          payload: whitehall_document,
```

Comment:
I think we put this outside the transaction originally so the payload and content_id will still get written to the database if the rest of the import fails.  It looks like that won't be the case anymore.

Reply:
Ah good to know - I'll move it back out 👍